bcg6es-d1e110/cod-Zy765t_bcg6es-d1e110.xml

```diff
@@ -83,7 +83,7 @@
           <lb ed="#P" n="21"/>tura, &amp; de illis subiectos suos sollicite praemonere; nam, sicut 
           <!--565.xml-->
           <pb ed="#P" n="565"/>
-          <lb ed="#P" n="1"/>dicit <ref xml:id="bcg6es-d1e110-Rd1e197" target="#bcg6es-d1e110-Qd1e207"><!-- add target when gloss entry is made -->Glossa Interlinearis</ref> super illud <ref xml:id="bcg6es-d1e110-Rd1e200" corresp="#bcg6es-d1e110-Qd1e202">Ezechiel. 3. <quote xml:id="bcg6es-d1e110-Qd1e202" source="http://scta.info/resource/ez3_17" synch="3-4">Speculatorem 
+          <lb ed="#P" n="1"/>dicit <ref xml:id="bcg6es-d1e110-Rd1e197" target="#bcg6es-d1e110-Qd1e207"> Glossa Interlinearis</ref> super illud <ref xml:id="bcg6es-d1e110-Rd1e200" corresp="#bcg6es-d1e110-Qd1e202">Ezechiel. 3. <quote xml:id="bcg6es-d1e110-Qd1e202" source="http://scta.info/resource/ez3_17" synch="3-4">Speculatorem 
```

Comment:
@CharlotteFeidicker  here on the `<ref>` `target="#bcg6es-d1e110-Qd1e207"` should be `corresp=#bcg6es-d1e110-Qd1e207` because this `<ref>` corresponds with the quote below. The corresponding quote below should then carry the `@source` and the value of the source should be the id of the text in the gloss ordinaria.

This write up might help explain things a bit: https://github.com/scta/scta-rdf-schema/blob/master/docs/proposals/quote-ref-relations.md

Reply:
I agree for the target/corresp - problem. Thanks for the link!
For the quotations, I thought that he would first quote the bible verse oft Ezechiel 3 and then quote the glossa of Ezechiel 3. Therfore I first quoted the bible verse and than the glossa. But you might be right, so that he first quotes the interlinear gloss and then the marginal gloss.